Update fi_strings.md

vanhavirta opened the pull request:
Translated by a native Finn -- hopefully this helps! The context for some strings was tricky to figure out, so I'll improve some case endings and singular/plural forms in the future.

--  @vanhavirta

androidseb commented:
Thank you for contributing to Map Marker translations :-)

I have a couple of questions for you:

* I usually add the translators names in the "thank you" section of Map Marker, is it OK if I mention you as "vanhavirta" in there?
* I see this PR is in "draft" and you made one more commit recently, should I wait a little more before using this work, or is it ready to go and I can pull in your translations as they are now?

vanhavirta commented:
Hi!

My pleasure, glad to be able to collaborate! Vanhavirta or Kimmo Virtanen
both work -- thank you for the mention! 🙌

The latest pull request from today is -- I think -- final for now. Since
there weren't separate strings for singular and plural (1 file vs 10
files), I added the plural case endings in parentheses.

Just a little tweak I did this morning, plus a couple of typos I fixed.

I'll start using Map Marker in Finnish and see if I can make it even better
in the future.

Thank you for a one of a kind app on the Play Store -- it really helps to
have an offline app for the most remote forest surveys I need to work on
(these days everything requires internet and a Google account). It's great
to have all my data locally on my device.

All the best,
Kimmo Virtanen (Premium member since yesterday)


On Sun, Mar 9, 2025, 17:21 androidseb ***@***.***> wrote:

> Thank you for contributing to Map Marker translations :-)
>
> I have a couple of questions for you:
>
>    - I usually add the translators names in the "thank you" section of
>    Map Marker, is it OK if I mention you as "vanhavirta" in there?
>    - I see this PR is in "draft" and you made one more commit recently,
>    should I wait a little more before using this work, or is it ready to go
>    and I can pull in your translations as they are now?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/androidseb/mapmarkerapptranslations/pull/17#issuecomment-2708911731>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BQIAHSSUACGAP62LWQA4S232TRL6DAVCNFSM6AAAAABYSYLQ4WVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMBYHEYTCNZTGE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> [image: androidseb]*androidseb* left a comment
> (androidseb/mapmarkerapptranslations#17)
> <https://github.com/androidseb/mapmarkerapptranslations/pull/17#issuecomment-2708911731>
>
> Thank you for contributing to Map Marker translations :-)
>
> I have a couple of questions for you:
>
>    - I usually add the translators names in the "thank you" section of
>    Map Marker, is it OK if I mention you as "vanhavirta" in there?
>    - I see this PR is in "draft" and you made one more commit recently,
>    should I wait a little more before using this work, or is it ready to go
>    and I can pull in your translations as they are now?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/androidseb/mapmarkerapptranslations/pull/17#issuecomment-2708911731>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BQIAHSSUACGAP62LWQA4S232TRL6DAVCNFSM6AAAAABYSYLQ4WVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMBYHEYTCNZTGE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

androidseb commented:
Thanks, I've merged your changes, I'll work on integrating them into Map Marker now, I'll let you know once the changes are applied.

androidseb commented:
@vanhavirta I reviewed your translations and integrated them into Map Marker's code, you can now see the changes in the latest beta version `3.12.0-beta-747` which will be available on Android and iOS once the builds have been processed, and is already available on [web](https://www.mapmarker.app/webapp_beta/)): if you navigate to "about" > "special thanks", you'll see your name in the "thank you" section :-).

Note that after reviewing your translations, I made one amend: I changed `settings_credits_option_title` from "Kartta-aineisto" to "Tekijätiedot", since the credits screen is not only about map data attributions, but also licenses. Please let me know if you think this needs to be adjusted, thanks.

Finnish is now 98% human-translated, thank you for your contribution :rocket:

vanhavirta commented:
Great, that amendment works too! "Tekijätiedot" literally means
"information about the creators or makers of the software" in Finnish, and
since the page includes credits for the (open) source maps being used and
not about you, the creator, I went with "Kartta-aineisto", meaning "map
data" or "map sources" 😎

On Sun, Mar 9, 2025, 19:04 androidseb ***@***.***> wrote:

> @vanhavirta <https://github.com/vanhavirta> I reviewed your translations
> and integrated them into Map Marker's code, you can now see the changes in
> the latest beta version 3.12.0-beta-747 which will be available on
> Android and iOS once the builds have been processed, and is already
> available on web <https://www.mapmarker.app/webapp_beta/>): if you
> navigate to "about" > "special thanks", you'll see your name in the "thank
> you" section :-).
>
> Note that after reviewing your translations, I made one amend: I changed
> settings_credits_option_title from "Kartta-aineisto" to "Tekijätiedot",
> since the credits screen is not only about map data attributions, but also
> licenses. Please let me know if you think this needs to be adjusted, thanks.
>
> Finnish is now 98% human-translated, thank you for your contribution 🚀
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/androidseb/mapmarkerapptranslations/pull/17#issuecomment-2708963670>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BQIAHSXDZPWESMTJ3X766CD2TRYBNAVCNFSM6AAAAABYSYLQ4WVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMBYHE3DGNRXGA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: androidseb]*androidseb* left a comment
> (androidseb/mapmarkerapptranslations#17)
> <https://github.com/androidseb/mapmarkerapptranslations/pull/17#issuecomment-2708963670>
>
> @vanhavirta <https://github.com/vanhavirta> I reviewed your translations
> and integrated them into Map Marker's code, you can now see the changes in
> the latest beta version 3.12.0-beta-747 which will be available on
> Android and iOS once the builds have been processed, and is already
> available on web <https://www.mapmarker.app/webapp_beta/>): if you
> navigate to "about" > "special thanks", you'll see your name in the "thank
> you" section :-).
>
> Note that after reviewing your translations, I made one amend: I changed
> settings_credits_option_title from "Kartta-aineisto" to "Tekijätiedot",
> since the credits screen is not only about map data attributions, but also
> licenses. Please let me know if you think this needs to be adjusted, thanks.
>
> Finnish is now 98% human-translated, thank you for your contribution 🚀
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/androidseb/mapmarkerapptranslations/pull/17#issuecomment-2708963670>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BQIAHSXDZPWESMTJ3X766CD2TRYBNAVCNFSM6AAAAABYSYLQ4WVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMBYHE3DGNRXGA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
   Kimmo Virtanen
   [http://vanhavirta.fi/]
   [http://www.facebook.com/vanhavirta <https://www.facebook.com/vanhavirta>
]
